src/templates/styles.module.scss

```diff
@@ -0,0 +1,192 @@
+
+@import '../theme/variables.scss';
+
+.header {
+  display: flex;
+  justify-content: space-around;
+  align-items: stretch;
+  flex-direction: column;
+  margin: 0 1rem 5rem 1rem;
+  border-bottom: 2px #DADADA solid;
+  height: 25rem; 
+  @media only screen and (min-width: $tablet) {
```

Comment:
this CSS needs to be cleaned up. Images do not alternate in detail view (only in list view on home page)